config.yaml

```diff
@@ -5,15 +5,15 @@ database:
   max_pool_size: 200
   replica_set: null
   username: mongoadmin
-  password: mongoadminsecret
+  password: "" # Set via BOOM_DATABASE__PASSWORD environment variable
   srv: false
 api:
   auth:
-    secret_key: "1234" # Secret key for JWT authentication, change it to a strong secret
-    token_expiration: 0 # jwt token expiration in seconds, 0 to disable. Should NOT be 0 in production
-    admin_username: admin # Admin username for the API
-    admin_password: adminsecret # Admin password for the API
-    admin_email: admin@example.com # Admin email for the API
+    secret_key: "" # Set via BOOM_API__AUTH__SECRET_KEY environment variable
+    token_expiration: 604800 # JWT token expiration in seconds (7 days) -- must be > 0
```

Comment:
any reason for that change? I actually find it quite annoying in a dev setup to not have 0 as the value. Then all the postman/insomnia/... kind of setup where you test API calls break weekly...

Reply:
Isn't it just as easy to store a username/password with those tools? My goal is to make it harder to deploy with an less secure config.